config/production.json

```diff
@@ -6,7 +6,7 @@
   },
   "logger": {
     "sendMailOnError": {
-      "to": "nicolas.dupont@beta.gouv.fr",
+      "to": "matti.schneider@beta.gouv.fr",
```

Comment:
I've just noticed this configuration option thanks to this PR. I don't see it documented in the "configuration" part of the README, https://github.com/ambanum/CGUs#configuring.

Is it documented somewhere else (I probably missed it)? Same for the extra environment options for `SMTP_*` variables.

If this is not the case, I can write a PR about these settings if you want (in the place that works best for you).

Reply:
Lovely, thanks for offering! Any help on improving the documentation would be welcome 😃

We should have updated the “Configuring” section of the README in #112 to reflect this new option 😞  This is where this documentation should go.

If you could document the environment variables there as well, perhaps splitting both parts in subtitles (`#### Config file` / `#### Environment variables`) and documenting the `HTTP*_PROXY` environment variables you added in #32, it would be awesome!